_posts/2022-10-13-septemeber-rubygems-updates.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Septemeber 2022 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in September.
+
+## RubyGems News
+
+This month in RubyGems, we released RubyGems [v3.3.22](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3322--2022-09-07) and Bundler [v2.3.22](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2322-september-7-2022).
+
+The following are the main improvements shipped during this month (see the changelog for more improvements and fixes):
+
+- removed `auto-sudo` feature for installing gems - [#5888](https://github.com/rubygems/rubygems/pull/5888).
```

Comment:
This one has not yet been released. Maybe I'd move it to a final paragraph after this list where I also want to mention working on improving our resolver, since that's most of what I did last month.